Misc/NEWS.d/next/Documentation/2020-02-19-11-13-47.bpo-17422.g7_9zz.rst

```diff
@@ -0,0 +1 @@
+The language reference now specifies restrictions on class namespaces.
```

Comment:
Since this isn't entirely your work @ananthan-123, let's credit Ethan here:

```suggestion
The language reference now specifies restrictions on class namespaces.  Adapted from a patch by Ethan Furman.
```